build/yamls/antrea-aks.yml

```diff
@@ -1371,6 +1371,15 @@ data:
     # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
     # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
     #kubeAPIServerOverride: ""
+
+    # Comma-separated list of Cipher Suites. If omitted, the default Go Cipher Suites will be used.
+    # https://golang.org/pkg/crypto/tls/#pkg-constants
+    # Note that TLS1.3 Cipher Suites cannot be explicitly listed here. Actually, the apiserver will
```

Comment:
How about:

Note that TLS1.3 Cipher Suites cannot be added to the list. But the apiserver will always prefer TLS1.3 Cipher Suites whenever possible.

Reply:
Updated.